cmd/activator/main.go

```diff
@@ -241,6 +245,22 @@ func main() {
 		}(name, server)
 	}
 
+	// TODO: Use configmap to enable tls mode.
+	if true {
+		tlsServers := map[string]*http.Server{
+			"https": pkgnet.NewServer(":"+strconv.Itoa(networking.BackendHTTPSPort), ah),
```

Comment:
Should we call [http2](https://pkg.go.dev/golang.org/x/net/http2#ConfigureServer) specific methods to enable  tls support?

Reply:
No, I think `ListenAndServeTLS` + http2 server handler is fine.

Reply:
Yeah verified it normal handler is enough.